niworkflows/common/report.py

```diff
@@ -8,6 +8,13 @@
 import jinja2
 from pkg_resources import resource_filename as pkgrf
 
+from nipype.interfaces.base import File, traits
+
+class ReportCapableInputSpec(object):
+    generate_report = traits.Bool(
+        desc="Set to true to enable report generation for node"
+    )
+
```

Comment:
Oscar added these to his input spec, which I quite like:

`
generate_report = traits.Bool(
        False, usedefault=True, desc='enable report generation')
html_report = File('viz-report.html', usedefault=True, desc='visual report file')
`

What do you think about adding these to `ReportCapableInputSpec`?

For consistency's sake I would prefer to call it `ReportCapableInterfaceInputSpec`. But I'll leave it up to you since that would be a monstrously long class name.